invenio_app_ils/records_relations/retriever.py

```diff
@@ -224,14 +229,14 @@ def get(self):
         return relations
 
 
-def get_relations(record):
+def get_relations(record, sort_by=None):
```

Comment:
```suggestion
def get_relations(record, **kwargs):
```
and also inject `kwargs` in all the `.get()` methods